R/teal_slice.R

```diff
@@ -481,50 +520,58 @@ c.teal_slices <- function(...) {
   )
 }
 
+#' @rdname teal_slice
+#' @keywords internal
+#'
+slices_to_list <- function(tss) {
```

Comment:
`slices_to_list` is ONLY used within `format.teal_slices`, these actions can be moved there. It would play well with my suggestion in `format.teal_slices`.

Reply:
yeah, but this function has a potential to be used in many more places in the future. it's not that custom as `slices_param_name`. for now it can be moved to `format.teal_slices` but I can predict we will take it out soon enough because has a potential to be used in other places of the code base

Reply:
fixed by chlebowa in next commits